packages/k8s/src/hooks/prepare-job.ts

```diff
@@ -229,6 +229,18 @@ export function createContainerSpec(
     }
   }
 
+  podContainer.env.push({
```

Comment:
What if `podContainer.env['GITHUB_ACTIONS']` already exists? Should we manage it as we do with `CI` to prevent duplication?